cli/errors.go

```diff
@@ -65,14 +70,28 @@ func (RootCmd) errorExample() *clibase.Cmd {
 			// A multi-error
 			{
 				Use: "multi-error",
+				Handler: func(inv *clibase.Invocation) error {
```

Comment:
should we hide this command? There's no reason a user would ever need to use this.

Reply:
It is hidden up the stack: https://github.com/coder/coder/blob/9656c7a0fea64b482bcf22c44370aeb1898cc1e2/cli/exp.go#L12-L12